gitlab/integration_test.go

```diff
@@ -183,6 +183,7 @@ var _ = Describe("GitLab Provider", func() {
 		testSharedOrgRepoName string = "testsharedorgrepo"
 		testOrgRepoName       string = "testorgrepo"
 		testRepoName          string = "testrepo"
+		testTreeRepoName      string = "testrepotree"
```

Comment:
Should we remove this in the cleanup step too?

Reply:
we actually clean because we genarate a random one with `test-repo-tree` prefix. But yes renamed here to follow the randomization names.